app/src/main/res/values-es/strings.xml

```diff
@@ -15,6 +15,9 @@
     <string name="error_authentication">Usuario no validado, contacte a su administrador</string>
     <string name="error_response_messages">No se pudieron mostrar los mensajes</string>
     <string name="retry">Reintentar</string>
+    <string name="text_description">Descripcion</string>
+    <string name="text_date">Fecha</string>
+    <string name="image_description">ImagenEstado</string>
```

Comment:
What is this supposed to be?

Reply:
for the initial text for the view

Reply:
If it's the deliverable status, then write `Deliverable Status Indicator` or something more descriptive. This is important for accessibility.

Reply:
Please address this